### ISSUE TITLE
Fix the Sign up as Doctor page

### ISSUE THREAD

**Issue by AshutoshRath1612** [CareerDevelopmentHub/MedKit#9]
When user select on the option “Sign up as Doctor” in “/register” page the page navigate to “/registerdoctor” page where it should have the following fields
• Name
• Email
• Password
• Confirm Password
• Age
• Gender
• Contact No.
• Specialist IN
• Years of Experience

**Comment by Jeevan-Kiran-Lenka** [comment]
As of now, it shows 
![Screenshot from 2022-10-08 13-59-50](https://user-images.githubusercontent.com/53392413/194698277-8361ffab-86a9-41ea-ac30-e5a95745fe4e.png)
for Google Sign In

and for Email Registration, it shows

![Screenshot from 2022-10-08 14-04-40](https://user-images.githubusercontent.com/53392413/194698341-6f5de1c5-762d-4334-a19a-5c777e9594ec.png)

Am I correct?

**Comment by AshutoshRath1612** [comment]
@Jeevan-Kiran-Lenka  yeah because i only have uploaded the frontend part. Just add the fields in signup page thats it.
